helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/model/AbstractTestClusterModel.java

```diff
@@ -116,11 +115,11 @@ protected ResourceControllerDataProvider setupClusterDataCache() throws IOExcept
     // 4. Mock two resources, each with 2 partitions on the default instance.
     // The instance will have the following partitions assigned
     // Resource 1:
-    //          partition 1 - MASTER
-    //          partition 2 - SLAVE
+    // partition 1 - MASTER
```

Comment:
Please try not to change it.
Or fill the line with "--------------"
Otherwise it becomes less readable.